metricbeat/module/mongodb/dbstats/dbstats.go

```diff
@@ -0,0 +1,89 @@
+package dbstats
+
+import (
+	"errors"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"github.com/elastic/beats/metricbeat/mb"
+	"github.com/elastic/beats/metricbeat/module/mongodb"
+	"gopkg.in/mgo.v2"
```

Comment:
Minor detail: We normally put this in between the general and the beats imports, so it is:

```
	"errors"

	"gopkg.in/mgo.v2"
	
	"github.com/elastic/beats/libbeat/common"
	"github.com/elastic/beats/libbeat/logp"
	"github.com/elastic/beats/metricbeat/mb"
	"github.com/elastic/beats/metricbeat/module/mongodb"
```